content/collections/workflow/en/experiment-templates.md

```diff
@@ -0,0 +1,140 @@
+---
+id: 8f9e2a1c-5d7b-4e3f-9c8a-2b1d6f4e8a3c
+blueprint: workflow
+title: 'Experiment templates'
+this_article_will_help_you:
+  - 'Understand what experiment templates are and when to use them'
+  - 'Create experiment templates from existing experiments'
+  - 'Use templates to quickly create new experiments'
+  - 'Manage and update your experiment templates'
+landing: false
+exclude_from_sitemap: false
+---
+
+Experiment templates capture the configuration of an experiment. Templates include goals, metrics, audience targeting, statistical preferences, and variants so you can reuse it for future experiments. Templates are particularly useful when you:
+
+- Run similar experiments across different features or time periods.
+- Want to enforce consistent experiment standards across your team.
+- Need to quickly launch new experiments with proven configurations.
+- Test variations of the same feature with different audiences.
+
+Templates are only applicable to Web Experiments.
```

Comment:
templates are not supported on web experiment. They are only supported on feature experiment